lms/templates/components/header/header.underscore

```diff
@@ -2,15 +2,15 @@
     <div class="page-header-main">
         <div class="sr-is-focusable" tabindex="-1"></div>
         <% if (breadcrumbs !== null && breadcrumbs.length > 0) { %>
-            <nav class="breadcrumbs" aria-label="<%- nav_aria_label %>">
+            <nav class="breadcrumbs" aria-label="<%= HtmlUtils.ensureHtml(nav_aria_label) %>">
             <% _.each(breadcrumbs, function (breadcrumb) { %>
-                <a class="nav-item" href="<%= breadcrumb.url %>"><%- breadcrumb.title %></a>
+                <a class="nav-item" href="<%= breadcrumb.url %>"><%= HtmlUtils.ensureHtml(breadcrumb.title) %></a>
```

Comment:
There were situations where titles were allowed to pass HTML. It seemed more flexible to support both in a generic header template.

Again, I wasn't sure that HTML escaping URLs was a good thing to be doing.